addons/danger/functions/fnc_tacticsSuppress.sqf

```diff
@@ -22,6 +22,15 @@ params ["_unit", "_target", ["_units", []], ["_delay", 17]];
 // find target
 _target = _target call CBA_fnc_getPos;
 
+// check target distance
+private _targetASL = ATLtoASL _target;
+
+// adjust pos
+private _vis = lineIntersectsSurfaces [(eyePos _unit) vectorAdd [0, 0, 15], _targetASL vectorAdd [0, 0, 2.5], _unit, objNull, true, 1, "GEOM", "VIEW"];
+if (_vis isNotEqualTo [] && {((_vis select 0) select 0) vectorDistance _targetASL > 143}) exitWith {
```

Comment:
use vectorDistanceSqr instead and check for 143*143 (20449). with that, there is no square root calculation
```suggestion
// 20449 = 143*143
if (_vis isNotEqualTo [] && {((_vis select 0) select 0) vectorDistanceSqr _targetASL > 20449}) exitWith {
```